docs/src/pages/components/screenreaderonly.mdx

```diff
@@ -0,0 +1,30 @@
+---
+name: Screen reader only
```

Comment:
What is this `name` field used for?  Other .mdx files have had this lower-case and singleworded, afaik.

Reply:
the idea is to use it to create the page title. i forgot to do that. i'll get a pr together for it